### PR TITLE
AER-2686 Add filterables to color labels legend

### PR DESCRIPTION
**Pull request by BertScholten**

### CODE REVIEW

**Review by JornC — APPROVED** [review]
ACK

File: gwt-shared-geo-common/src/main/java/nl/aerius/geo/domain/legend/ColorLabelsLegend.java [hunk 1]
```diff
@@ -34,6 +34,7 @@ public class ColorLabelsLegend implements Legend {
   private String outlineColor;
   private Integer[] iconSizes;
   private LegendType icon;
+  private Object[] filterables;
```
Comment (Hilbrand) [inline gwt-shared-geo-common/src/main/java/nl/aerius/geo/domain/legend/ColorLabelsLegend.java:37]: Was this tested? GWT doesn't like `Object` fields... and it seems to kill the startup of Calculator..

Comment (BertScholten) [reply gwt-shared-geo-common/src/main/java/nl/aerius/geo/domain/legend/ColorLabelsLegend.java:37]: Locally with codeserver it seemed to work fine. Is this with a deployed version?

Comment (Hilbrand) [reply gwt-shared-geo-common/src/main/java/nl/aerius/geo/domain/legend/ColorLabelsLegend.java:37]: Yes deployed. My pr built won't start up.

**Pull request merged**